browser_use/browser/types.py

```diff
@@ -0,0 +1,717 @@
+	)
+
+	# --- Page load/wait timings ---
+	minimum_wait_page_load_time: float = Field(default=0.25, description='Minimum time to wait before capturing page state.')
+	wait_for_network_idle_page_load_time: float = Field(default=0.5, description='Time to wait for network idle.')
+	maximum_wait_page_load_time: float = Field(default=5.0, description='Maximum time to wait for page load.')
+	wait_between_actions: float = Field(default=0.5, description='Time to wait between actions.')
+
+	# --- UI/viewport/DOM ---
+	include_dynamic_attributes: bool = Field(default=True, description='Include dynamic attributes in selectors.')
+	highlight_elements: bool = Field(default=True, description='Highlight interactive elements on the page.')
+	viewport_expansion: int = Field(default=500, description='Viewport expansion in pixels for LLM context.')
+
+	profile_directory: str = 'Default'  # e.g. 'Profile 1', 'Profile 2', 'Custom Profile', etc.
+
+	save_recording_path: str | None = Field(default=None, description='Directory for video recordings.')
+	save_downloads_path: str | None = Field(default=None, description='Directory for saving downloads.')
+	save_har_path: str | None = Field(default=None, description='Directory for saving HAR files.')
+	trace_path: str | None = Field(default=None, description='Directory for saving trace files.')
+
+	# extension_ids_to_preinstall: list[str] = Field(
+	# 	default_factory=list, description='List of Chrome extension IDs to preinstall.'
+	# )
+	# extensions_dir: Path = Field(
+	# 	default_factory=lambda: Path('~/.config/browseruse/cache/extensions').expanduser(),
+	# 	description='Directory containing .crx extension files.',
+	# )
+
+	# # --- File paths ---
+	# downloads_dir: Path = Field(
+	# 	default_factory=lambda: Path('~/.config/browseruse/profiles/default/downloads').expanduser(),
+	# 	description='Directory for downloads.',
+	# )
+	# uploads_dir: Path | None = Field(default=None, description='Directory for uploads (defaults to downloads_dir if not set).')
+
+	def __repr__(self) -> str:
+		return f'BrowserProfile(label={self.label})'
+
+	def __str__(self) -> str:
+		return repr(self)
+
+	def get_args(self) -> list[str]:
+		return BrowserLaunchArgs.args_as_list(  # convert back to ['--arg=value', '--arg', '--arg=value', ...]
+			BrowserLaunchArgs.args_as_dict(  # uniquify via dict {'arg': 'value', 'arg2': 'value2', ...}
+				[
+					*CHROME_DEFAULT_ARGS,
+					*self.args,
+					f'--profile-directory={self.profile_directory}',
+					*(CHROME_DOCKER_ARGS if IN_DOCKER else []),
+					*(CHROME_HEADLESS_ARGS if self.headless else []),
+					*(CHROME_DISABLE_SECURITY_ARGS if self.disable_security else []),
+					*(CHROME_DETERMINISTIC_RENDERING_ARGS if self.deterministic_rendering else []),
+					*(
+						[f'--window-size={self.window_size["height"]},{self.window_size["width"]}']
+						if self.window_size
+						else (['--start-maximized' if not self.headless else []])
```

Comment:
This should be :
```python
else (['--start-maximized'] if not self.headless else [])
```
otherwise getting below error: 
```
key, value, *_ = [*arg.split('=', 1), '', '', '']
E   AttributeError: 'list' object has no attribute 'split'
```
(because we are getting [[]]  # list of list!)